src/io/file_system.ts

```diff
@@ -110,39 +112,57 @@ export class NodeFileSystem implements tfc.io.IOHandler {
       };
     }
   }
-
   async load(): Promise<tfc.io.ModelArtifacts> {
-    if (Array.isArray(this.path)) {
-      throw new Error('Loading from multiple paths is not supported yet.');
-      // TODO(cais, nkreeger): Implement this. See
-      //   https://github.com/tensorflow/tfjs/issues/343
-    }
+    return Array.isArray(this.path) ? this.loadBinaryModel() :
+                                      this.loadJSONModel();
+  }
 
-    const info = await stat(this.path).catch(doesNotExistHandler('Path'));
+  protected async loadBinaryModel(): Promise<tfc.io.ModelArtifacts> {
+    const topology =
+        await stat(this.path[0]).catch(doesNotExistHandler('Path'));
```

Comment:
`doesNotExistHandler()` takes a string to specify what could not be found. Probably helpful to clarify that `'Path'` is really `'topology path'`.

Same below for `weightManifest`: